docs/4-At-The-Internship/getting_most_out_of_internship.md

```diff
@@ -0,0 +1,19 @@
+## Getting the most out of your internship
+
```

Comment:
this file is a bit odd to me in general since this whole folder is about "getting the most out of your internship" not just this file. I think it'd be better to make separate files about "exploring new cities" and "developing technical skills"